wtr-utils.js

```diff
@@ -155,14 +155,9 @@ const testRunnerHtml = (testFramework) => `
   </html>
 `;
 
-const getScreenshotFileName = ({ browser, name }, type, diff) => {
+const getScreenshotFileName = ({ name }, type, diff) => {
   const [component, test] = name.split(':');
-  return path.join(
-    browser.replace('Windows 10 ', '').replace(' latest', ''),
-    type,
-    component,
-    diff ? `${test}-diff` : test
-  );
+  return path.join('chrome', type, component, diff ? `${test}-diff` : test);
```

Comment:
Is that just a refactoring or it returns a different path?

Reply:
Previously this was checking for `latest` and allowed for other browser than Chrome.

As we had some parts hardcoded anyways, I decided to use the full name instead to make it clear.

Reply:
The artifacts are not affected by this change.